mlflow/projects/docker.py

```diff
@@ -24,6 +24,28 @@
 _PROJECT_TAR_ARCHIVE_NAME = "mlflow-project-docker-build-context"
 
 
+def _parse_user_env_vars(user_env_vars):
+    env_vars = {}
+    if user_env_vars is not None:
+        for user_entry in user_env_vars:
+            if isinstance(user_entry, list):
+                # User has defined a new environment variable for the docker environment
+                env_vars[user_entry[0]] = user_entry[1]
+            else:
+                # User wants to copy an environment variable from system environment
+                system_var = os.environ.get(user_entry)
```

Comment:
What is the data type of `user_env_vars` here? Can you provide an example of what it might look like? That will help maintainers to understand the logic.